_rules/SC2-4-6-descriptive-labels.md

```diff
@@ -21,27 +21,38 @@ authors:
 
 ### Applicability
 
-This rule applies to any HTML `label` element or other element referenced by `aria-labelledby` that:
+This rule applies to any element that:
+- has one of the following [semantic roles](#semantic-role): `checkbox`, `combobox` (`select` elements), `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider`, `spinbutton`, `switch` and `textbox`, and
+- is [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree), and 
+- has an [accessible name](#accessible-name) or is the [labeled control](https://www.w3.org/TR/html/sec-forms.html#labeled-control) of a `label` element that is either [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
I think there must a note on the accessible name computation, or a least, a link in the background section : https://www.w3.org/TR/html-aam-1.0/#accessible-name-and-description-computation

Reply:
@audreymaniez, it is part of the [accessible name](https://act-rules.github.io/glossary/#accessible-name) definition. Isn't that enough?

Reply:
han you're totally right! no I think that enough!